### DISCUSSION TITLE
How to make my package actually public

### DISCUSSION THREAD

**Discussion by lwahonen** [community/community#66861] [Packages]
### Select Topic Area

Product Feedback

### Body

I've changed the repo to be public in https://github.com/lwahonen/ref-napi/settings/access and I've changed the package to be public in https://github.com/users/lwahonen/packages/npm/ref-napi/settings

But npm still reliably fails for users with npm ERR! 401 Unauthorized - GET https://npm.pkg.github.com/download/@lwahonen/ref-napi/4.0.11/e36233cfa2e6dcf2f1e6e02f1dbe1039f0379e5a - authentication token not provided

**Comment by ryanobjc** [comment]
Have you figured it out yet?  You can't.  That isn't a feature they support, nor will they support apparently!
